docs/message_v3.md

```diff
@@ -0,0 +1,154 @@
+# Tox MessageV3 (alias Hubble telescope glasses)
+
+
+proposal to transpartently patch current text messages to fix double messages and missed messages (and to support history sync in the future)
+
+
+new defines:
+------------
+
+```
+#define TOX_MAX_MESSAGE_LENGTH         (1372)
+#define TOX_MSGV3_MSGID_LENGTH         (32)
+#define TOX_MSGV3_TIMESTAMP_LENGTH     (4)
```

Comment:
https://en.wikipedia.org/wiki/Year_2038_problem

Reply:
i know. but whats the fix? using 64bits is taking 8 bytes.
is there a better solution that 64bits?

Reply:
Unsigned 32-bits should work until year 2106.

If extended to 5 bytes = 40bits, it could store a range of about 34865 years.